src/build/__main__.py

```diff
@@ -17,6 +17,14 @@
 from build.env import IsolatedEnvBuilder
 
 
+try:
+    import colorama
+except ImportError:
+    pass
+else:
+    colorama.init()  # fix colors on windows
```

Comment:
This should probably be done in `main()`, not on module import, even if that module is `__main__`.

Reply:
It should actually be done on `build_package`, but that may run many times. I think this approach is *fine*, but if you really think it would be worth to change, I can do that.

Reply:
I don't feel too strongly about it, more a matter of principle, that code shouldn't run on import - let's keep it as is.

Reply:
I also vote to do it in the main method... I don't see why it would it be done in build_package?

Reply:
If I do the following, colors will be broken.

```python
import build.__main__

build.__main__.build_package(...)
```

We already set the `showwarning` on import, so we have precedent, if we choose to not run this on import, we should move that too.
We can also change the implementation so that `build_package` to not call `_error`, but rather raise an error, and then we would call `_error` in `main`.

I would go for the PR as-is, and then one of you can make a followup one fixing what you want.